pkg/pow/solutionHandler.go

```diff
@@ -80,7 +80,7 @@ func (s *SolutionHandler) Submit(ctx context.Context, result *Result) (*types.Tr
 		} else {
 			value, err = hexutil.DecodeBig(string(val))
 			if err != nil {
-				if challenge.RequestIDs[i].Uint64() > 53 {
+				if challenge.RequestIDs[i].Uint64() > 56 {
```

Comment:
to avoid changing this all the time we can add in the `tracker` package
```
func MaxPSRID() uint64 {
	var maxID int
	for id := range PSRs {
		if id > maxID {
			maxID = id
		}
	}
	return uint64(maxID)
}
```
and then this will become

```suggestion
				if challenge.RequestIDs[i].Uint64() > tracker.MaxPSRID() {
```

btw can you double check the logic here as it seems strange, why does it run this check only when there is an error?

Reply:
If you have a value, the requestID obviously isn't above the max psr id...so not need checking

Reply:
Yep, I see your point, but we still need to make the errors and checks a bit more clear, can be a diff PR, but should have it in mind.